docs/cli.md

```diff
@@ -58,12 +59,19 @@ Options:
   --num-dp-cores=<num>  Number of data plane cores [default: 4].
   --num-cp-cores=<num>  Number of control plane cores [default: 1].
   --pool=<pool>         Pool name: either infra, controlplane or dataplane.
+  --cp-mode=<mode>      Control plane core allocation mode. Possible modes:
+                        packed and spread [default: packed].
+  --dp-mode=<mode>      Data plane core allocation mode. Possible modes:
+                        packed and spread [default: packed].
   --publish             Whether to publish reports to the Kubernetes
                         API server.
   --pull-secret=<name>  Name of secret used for pulling Docker images from
                         restricted Docker registry.
   --saname=<name>       ServiceAccount name to pass
                         [default: cmk-serviceaccount].
+  --socket-id=<num>     ID of socket where allocated core should come from.
```

Comment:
Please pardon the late review, but curious whether this option is necessary. Isn't it equivalent to creating multiple pools? One advantage to the previous structure was that topology was confined to the initialization phase which allowed isolate to be very simple to implement and test.